### ISSUE TITLE
GeoJSONs compatible with EU Information System

### ISSUE THREAD

**Issue by jo-spek** [google/ground-platform#2123]
Splitting of from https://github.com/google/ground-platform/issues/2013:

Once GeoJSON download of job data is possible, next step would be to implement **columns** in the GeoJSON file that can automatically be read by the EU Information System for Due Diligence Reporting. [Requirements here](https://circabc.europa.eu/ui/group/34861680-e799-4d7c-bbad-da83c45da458/library/adebc1da-4e60-4479-83dd-ee344607c6f9/details?download=true).

The column titles need to be **precisely**:
 
- "ProducerCountry" (**mandatory** ISO2 country code) --> could be automatically calculated from the geometry or enquired in the survey with a dropdown menu country selection.
- "Area" (**mandatory** hectareage of the geometry) --> can be automatically calculated for polygons and enquired from the survey for point data. If not provided for point data, EU-Information will still ingest and default to 4 hectares for points. 
- "ProducerName" (**optional**) --> to be enquired in the survey. Not mandatory, but best to be set as field in survey design already.
- "ProductionPlace" (**optional**) --> could be automatically calculated from the geometry based on global admin-datasets like GAUL, AND/OR enquired from the survey the same way as the producer name.

When the columns are provided and filled with data, the EU Information System will automatically ingest and fill the fields as shown below:

![Image](https://github.com/user-attachments/assets/560a6287-d5e5-454a-baab-74af24b80b37)

**Comment by kenstershiro** [comment]
@rfontanarosa to check if all the data is already available

**Comment by kenstershiro** [comment]
Agreed to move to Icebox as most of this can be derived from existing data easily.

**Comment by jo-spek** [comment]
Whisp API currently returns "**Area**" and "**ProducerCountry**". Removing the prefix and including in the GeoJSON will make the export immediately ingestible by the EU Information system (however, they really could also provide that themselves, recognizing that area and producer country are easily detectable by GIS code.)

The two fields "**ProducerName**" and "**ProducerCountry**" that are non-mandatory but functional in the EU Information System, could be implemented through something as suggested in https://github.com/google/ground-platform/issues/769. Having a public "EUDR survey" that does that would be neat. 

Leaving this iceboxed for the time being.
